pylint/checkers/__init__.py

```diff
@@ -128,7 +132,7 @@ def table_lines_from_stats(
     return lines
 
 
-def initialize(linter):
+def initialize(linter: PyLinter) -> None:
     """Initialize linter with checkers in this package."""
     register_plugins(linter, __path__[0])
```

Comment:
Not related to the current MR but it looks a little too magical, maybe we should use hard coded values ?

Reply:
?

It works kinda nicely as we don't need to add any new checkers when we add them.

Reply:
Well, I'm saying that because I don't understand how it works 😄